dpgen2/exploration/selector/conf_filter.py

```diff
@@ -48,11 +70,19 @@ def add(
 
     def check(
         self,
-        conf: dpdata.System,
-    ) -> dpdata.System:
-        natoms = sum(conf["atom_numbs"])  # type: ignore
-        selected_idx = np.arange(conf.get_nframes())
+        ms: dpdata.MultiSystems,
+    ) -> dpdata.MultiSystems:
+        selected_idx = sum(
+            [[(i, j) for j in range(s.get_nframes())] for i, s in enumerate(ms)], []
+        )
```

Comment:
It is not clear for me why we need a `sum` here. You may also consider the suggestion by rabbitAI